docs/examples/junit4/generic/build.gradle

```diff
@@ -10,3 +10,5 @@ dependencies {
     testImplementation "org.seleniumhq.selenium:selenium-api:4.25.0"
     testImplementation 'org.assertj:assertj-core:3.26.3'
 }
+
+project.tasks.test.dependsOn jar
```

Comment:
This code looks suspicious. It st really required?

Reply:
I guess it might be caused by https://github.com/testcontainers/testcontainers-java/blob/63189c8678f81c8ef4c431e4468ee74640682654/gradle/shading.gradle#L60, so the way to fix it is to refactor `shading.gradle` rather than adding the dependency